src/shared/cloudformation/cloudformation.ts

```diff
@@ -284,4 +304,209 @@ export namespace CloudFormation {
             resource.Properties.Handler === handlerName
         )
     }
+
+    /**
+     * Parameter/Ref helper functions
+     */
+
+    /**
+     * Validates whether or not a property is a valid ref.
+     * @param property Property to validate
+     */
+    function isRef(property: string | number | object | undefined): boolean {
+        return (
+            typeof property === 'object' && Object.keys(property).length === 1 && Object.keys(property).includes('Ref')
+        )
+    }
+
+    /**
+     * Gets the string value for a property in a template.
+     * If the value is a Ref to a parameter, returns the default value of the ref; this may be undefined.
+     * Also returns undefined if the property is undefined or a number.
```

Comment:
if `validatePropertyType` later supports more types, then this doc will be a bit misleading. 

This makes it future proof (and more informative anyway):

```suggestion
     * Also returns undefined if the property is not a string.
```